lib/ImageProcessor.js

```diff
@@ -51,18 +51,21 @@ class ImageProcessor {
         const bucket = config.get("bucket");
         const jpegOptimizer = config.get("jpegOptimizer", "mozjpeg");
 
-        let promise = new Promise((resolve) => { resolve(); });
+        let promise = Promise.resolve();
         let processedImages = 0;
+        let removeAfterSuccess = false;
 
         if ( config.exists("backup") ) {
             const backup = config.get("backup");
             backup.acl = backup.acl || acl;
             backup.bucket = backup.bucket || bucket;
+            // If backup.move option is true, remove original object after completion
+            removeAfterSuccess = Boolean(backup.move);
```

Comment:
`Boolean(backup.move)` seems to be not the best approach: https://stackoverflow.com/questions/263965/how-can-i-convert-a-string-to-boolean-in-javascript